doc/api/http2.md

```diff
@@ -2598,3 +2598,16 @@ if the stream is closed.
 [Readable Stream]: stream.html#stream_class_stream_readable
 [`ServerRequest`]: #http2_class_server_request
 [`stream.pushStream()`]: #http2_stream-pushstream
+[`http2.Server`]: #http2_class_http2server
```

Comment:
Thank you 🥇
While you are here it would be nice if you could sort the whole definition block in "ASCIIbetical" order.